### ISSUE TITLE
Extension identifier label text has color contrast ratio less than 4.5:1.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode#108300]
**“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**

GitHub Tags: #A11yMAS; #A11yTCS; #MAC; #ColorContrast; #A11y_VSCode; #WCAG1.4.3; #A11ySev2; #AINotInScope;#CAITestPass_May2020;#Benchmark;#Visual Studio Code;#FTP;

### Environment Details:
Application Name: VS Code
Application Version: 1.48.0-insider
Commit: cfbd1999769f4f08dce29629fb92fdc0fac53829
Date: 2020-08-06T05:40:40.871Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina Version 10.15.6 (19G73)
Screen Reader: VoiceOver

### Repro Steps:
1. Open Visual studio code 
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button, the "Default Screen" will appear.
4. Navigate to "ms-azuretools.vscode-azurefunctions" text and verify if the color contrast ratio is less than 4.5:1 or not.

### Actual:
The "ms-azuretools.vscode-azurefunctions" text has color contrast ratio of 2.6:1 which is less than the standard color contrast ratio of 4.5:1 which is not appropriate as the users with low vision will have difficulty in reading the text on the screen.

### Expected:
The color contrast ratio of "ms-azuretools.vscode-azurefunctions" text should be greater than or equal to 4.5:1.

### User Impact:
If the color contrast ratio of the content present on the screen is less than 4.5:1 then the users with low vision will have difficulty in reading the data on the screen.

### Attachment link for Reference:
<img width="1400" alt="2280_A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_ColorContrast The  ms-azuretools vscode-azurefunctions  text has color contrast ratio less than 4 51" src="https://user-images.githubusercontent.com/52036772/89617823-8541aa80-d8a8-11ea-9f3d-b93149881fa7.png">
